### PR TITLE
Add The Scarlett - A Buildable Ship

### PR DESCRIPTION
**Pull request by IllusiveMan196**
Add mod, location, cleaning data

### CODE REVIEW

File: masterlist.yaml [hunk 1]
```diff
@@ -24908,3 +24908,11 @@ plugins:
   - name: 'MarkekrausExitCombatPower.esl'
     url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/58651/' ]
     msg: [ *obsolete ]
+
+  - name: 'TheScarlett.esp'
+    url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/2434/' ]
```
Comment (sibir-ine) [inline masterlist.yaml]: The location would benefit from a name:

```suggestion
    url:
      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/2434/'
        name: 'The Scarlett - A Buildable Ship'
```

**Review by sibir-ine — CHANGES_REQUESTED** [review]
This just needs to be rebased now.

**Comment by IllusiveMan196** [comment]
> This just needs to be rebased now.

Done.

**Review by sibir-ine — APPROVED** [review]
Thanks!

**Pull request merged**